udagram-api-feed/docker-compose.yml

```diff
@@ -1,16 +1,16 @@
-version: '3.8'
-services:
-  postgres:
-    image: postgres:latest
-    container_name: udagram-db
-    ports:
-      - "5432:5432"
-    environment:
-      POSTGRES_DB: udagram-db
-      POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: postgres
-    volumes:
-      - postgres-data:/var/lib/postgresql/data
-
-volumes:
+version: '3.8'
+services:
+  postgres:
+    image: postgres:latest
+    container_name: udagram-db
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_DB: udagram-db
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
```

Comment:
You could leverage env variables for the password, use and username. You can even substitute using ${ENV_VAR} if it's different. You could also use .env_files - https://docs.docker.com/compose/environment-variables/env-file/